docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig

```diff
@@ -0,0 +1,22 @@
+{% if variant == 'open' %}
+  {% set attributes = attributes.setAttribute('open', 'open') %}
+{% endif %}
+
+<details{{ attributes }}>
+  {%- if title -%}
```

Comment:
TIL: Whitespace control at the tag level. Neato. 
I've been tempted to add {% spaceless %} around the whole html.html.twig document but this is nice as well.

Reply:
Yeah i grabbed this template from `stable` theme and had to look that up.

Reply:
Oh, this is from stable? Can you not just extend that template here then?

Reply:
stable doesnt have blocks and i needed to wrap `description` and `children` in a wrapper.